Dockerfile

```diff
@@ -1,15 +1,32 @@
 # Choose your LLVM version
-ARG LLVM_VERSION=8.0
+ARG LLVM_VERSION=800
+ARG ARCH=amd64
+ARG DISTRO_BASE=ubuntu18.04
+ARG LIBRARIES=/opt/trailofbits/libraries
+ARG Z3_VERSION=4.7.1
+#this seems to be static for most Z3 Linux builds
+ARG Z3_OS_VERSION=ubuntu-16.04
+ARG Z3_ARCHIVE=z3-${Z3_VERSION}-x64-${Z3_OS_VERSION}
 
 # Run-time dependencies go here
-FROM ubuntu:18.04 as base
+FROM trailofbits/cxx-common:llvm${LLVM_VERSION}-${DISTRO_BASE}-${ARCH} as base
```

Comment:
Using `cxx-common` as a base here will result in a very large final image, since `cxx-common` is huge.

What is the end-goal of this Docker image? I thought we wanted to just package `rellic` by itself in the resulting Docker image

Reply:
Wouldn't we need cxx-common to run it?